src/containers/UserProfile/AboutUser.js

```diff
@@ -0,0 +1,156 @@
+import React, { Component } from "react";
+import SimpleReactValidator from "simple-react-validator";
+import ReactNotification from "react-notifications-component";
+import "react-notifications-component/dist/theme.css";
+import addNotification from "../../utilities/index";
+import * as auth from "../../services/Session";
+import API from "../../api/index";
+
+class AboutUser extends Component {
+  constructor(props) {
+    super(props);
+
+    this.validator = new SimpleReactValidator();
+    this.state = {
```

Comment:
if there is one field in the object you can write it in one line

Reply:
I have moved the field in one line.